aws_google_auth/google.py

```diff
@@ -127,6 +127,9 @@ def do_login(self):
 
         self.session.headers['Referer'] = sess.url
 
+        if "selectchallenge/" in sess.url:
```

Comment:
When does this occur?  Is it only to choose between SMS and Voice, or is that just the use-case you have at the moment.

I've been hunting for any Google docco on how this works, but might need to setup some different accounts to test the combinations.